packages/react-widgets/src/models/TableModel.js

```diff
@@ -19,3 +24,32 @@ function fromLocal(props) {
     sortByColumnType
   });
 }
+
+// @todo we need to cache res somehow otherwise we'll hit the API
```

Comment:
This could be probably well managed adding react-query as a new big but useful dep of the package to cache in the frontend. We can have that discussion in other moment

Reply:
https://tanstack.com/query/v4/docs/react/examples/react/pagination